test/core/templaters/base_test.py

```diff
@@ -36,131 +36,154 @@ def test__templater_raw():
     assert instr == str(outstr)
 
 
-SIMPLE_SOURCE_STR = "01234\n6789{{foo}}fo\nbarss"
```

Comment:
How carefully should I review the changes to this file? Could you possibly highlight (i.e. by adding PR comments) on the areas that need close attention? It looks like a mix of "real" changes and formatting/mechanical changes.

Reply:
I've had a quick run through - and _all of the changes to this file are "mechanical changes"_. The main thing that I'm changing though is that some tests used to use different portions of `raw_sliced` and `sliced_file` - which was causing more inconsistency issues. 

What I'm mainly doing in this upper section is making sure when a test uses one of these two examples that they get enough to make a _complete TemplatedFile_ - with filename, with raw slices, with templated slices and with strings for the source and templated files. The changes in this PR make it more important that a `TemplatedFile` is properly constructed - but in real use - that is always true (or at least _should_ be true), so we don't need test cases which cover incomplete construction. If they are incomplete - then the consistency checks will fire.